packages/common/src/services/audius-api-client/AudiusAPIClient.ts

```diff
@@ -1814,7 +1818,10 @@ export class AudiusAPIClient {
     pathType: PathType = PathType.VersionPath
   ) => {
     const formattedPath = this._formatPath(pathType, path)
-    return this._constructUrl(formattedPath, queryParams)
+    return this._constructUrl(formattedPath, {
```

Comment:
nice, this works or you can add `this.appName` inside of _constructUrl's function so callers dont need to worry about it?

Reply:
cool ya done